xml/security_fips.xml

```diff
@@ -57,6 +47,15 @@
     and is replacing 140-2. CMVP is no longer accepting modules for
     FIPS 140-2 testing, but only FIPS 140-3.
   </para>
+  <important>
+   <title>&sles; &productnumber; and FIPS 140-3</title>
+  <para>
+   &productname; &productnumber; is in the process of implementing the
```

Comment:
```suggestion
   &sles; &productnumber; is in the process of implementing the
```

Reply:
Let's use a consistent productname within the admonition. Decided to go for 'SUSE Linux Enterprise Server' specifically, based on the target version of PM-3612.